src/js/components/addData/AddDataMeta.jsx

```diff
@@ -220,6 +260,12 @@ export default class AddDataMeta extends React.Component {
                                 </div>
                             </div>
                         </div>
+                        <Modal onClose={this.closeModal} isOpen={this.state.showModal} buttons={
+                            <div>
```

Comment:
declare the button before the html and store it in a variable. then pass it in for the params.
eg.
```
let button = <button>Text</button>
...
<Modal buttons={button}/>```